Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -295,23 +296,37 @@ WarpXOpenPMDPlot::Init (openPMD::Access access, bool isBTD)
     std::string filepath = m_dirPrefix;
     GetFileName(filepath);
 
+    std::string useSteps = R"(
```

Comment:
With https://github.com/openPMD/openPMD-api/pull/855 merged, the logic that we want to trigger is a third variant of the `iterationEncoding`.

We set it tlike this:
```
m_Series.setIterationEncoding( IterationEncoding::variableBased );
```

and it implies `useStep`, so no JSON options are needed.

For more background, see the changes in the linked PR to `test/SerialIOTest.cpp ` and `test/ParallelIOTest.cpp`. (But the one-line I mentioned is basically it.)

We could generalize the option that you expose as `m_OneFilePerTS` to allow these three values now.

Reply:
OK, I updated and recompiled and will test it out. 

Looks like for adios, FileBased and VariableBased would be enough.  And the Boolean variable m_OneFilePerTS would do.